mock_discord.py

```
@@ -0,0 +1,189 @@
+        self.user = MockUser(user_id=111111111, name="TechFrenBot", bot=True)
+        self.guilds = [MockGuild()]
+        
+    async def close(self):
+        """Mock closing the client"""
+        print("[DEBUG]: Mock client closed")
+
+
+class MockAllowedMentions:
+    """Mock Discord AllowedMentions object"""
+    @staticmethod
+    def none():
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)